transportable-udfs-trino-plugin/src/main/java/com/linkedin/transport/trino/TransportConnectorFactory.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
+import io.airlift.log.Logger;
+import io.trino.spi.connector.Connector;
+import io.trino.spi.connector.ConnectorContext;
+import io.trino.spi.connector.ConnectorFactory;
+import io.trino.spi.function.FunctionId;
+import java.io.File;
+import java.io.FileFilter;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+import static java.util.Objects.*;
+
+
+public class TransportConnectorFactory implements ConnectorFactory {
+
+  private static final String TRANSPORT_UDF_MP = "/transport-udf-mp";
```

Comment:
Given Transport is open sourced, should this not refer Linkedin specific term?

Reply:
Good point, modify it to `transport-udf-repo`